gym/lib/gym/runner.rb

```diff
@@ -155,14 +159,27 @@ def move_ipa
       ipa_path
     end
 
+    # copys commandline tool binarys
+    def copy_commandline_tool
+      cmd_path = "Products/usr/local/bin/*"
+      if Gym.project.commandline_library?
+        cmd_path = "Products/usr/local/lib/*"
+      end
+      app_path = Dir[File.join(BuildCommandGenerator.archive_path, cmd_path)]
+      app_path.each do |f|
```

Comment:
Should we be expecting more than one executable or lib to be built here? In my experience building CLI apps in Xcode, I have only had one product in `Products/usr/local/bin/`, so I'm just curious under what circumstances this would be the case?

Also, would it be cool to change the name of the variable to `tool_path` or something just so it won't be confused with an "app" and goes along with the other naming conventions of `copy_commmandline_tool`?

And for one more pedantic comment, could we change this to be `copy_command_line_tool`?